setup.py

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/env python
 
 from setuptools import setup
+from setuptools import find_packages
 
 setup(name="notify",
       version="0.1",
       description="Notify service",
       url="https://github.com/seecloud/notify",
       author="<name>",
       author_email="<name>@mirantis.com",
-      packages=["notify"],
+      packages=find_packages(),
```

Comment:
It is better to exclude `tests` from the list of packages, otherwise `tests` will be shipped alongside with the `notify` package. It could be done like that:

```
     packages=find_packages(exclude=["tests*"])
```